src/cli.rs

```diff
@@ -329,6 +329,9 @@ pub(crate) struct LlvmCovOptions {
     pub(crate) ignore_filename_regex: Option<String>,
     // For debugging (unstable)
     pub(crate) disable_default_ignore_filename_regex: bool,
+    // For debugging (unstable)
+    /// Disable skipping of files from vendored sources.
+    pub(crate) disable_default_ignore_vendor: bool,
```

Comment:
Are there any use cases where this flag is needed for purposes other than testing this implementation? If not, I would prefer not to add the flag (i.e., use disable_default_ignore_filename_regex for this) and keep things simple.